lib/cpu.js

```diff
@@ -16,7 +16,7 @@ var getUsage = function(cb) {
   });
 };
 
-var alreadySendingToKey = {};
+var checkInterval = {};
 
 module.exports = function(influxclient) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Function has a complexity of 22 which is greater than 20 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AFunctionComplexity)